test/extended/util/image_helpers.go

```diff
@@ -61,16 +70,28 @@ func VerifyImagesDifferent(comp1, comp2, strategy string) {
 
 //WaitForBuild is a wrapper for WaitForABuild in this package that takes in an oc/cli client; some ginkgo based debug along with ginkgo error checking
 func WaitForBuild(context, buildName string, oc *CLI) {
-	g.By(fmt.Sprintf("\n%s %s:   waiting for %s", time.Now().Format(time.RFC850), context, buildName))
+	g.By(fmt.Sprintf("%s:   waiting for %s", context, buildName))
 	WaitForABuild(oc.REST().Builds(oc.Namespace()), buildName, CheckBuildSuccessFunc, CheckBuildFailedFunc)
 	// do not care if build returned an error ... entirely possible ... we only check if the image was updated or left the same appropriately
-	g.By(fmt.Sprintf("\n%s %s   done waiting for %s", time.Now().Format(time.RFC850), context, buildName))
+	g.By(fmt.Sprintf("%s   done waiting for %s", context, buildName))
 }
 
-//StartBuild creates a build config from the supplied json file (not a template) and then starts a build, using the supplied oc/cli client for both operations; ginkgo error checking included
-func StartBuild(jsonFile, buildPrefix string, oc *CLI) {
+//StartBuildFromJSON creates a build config from the supplied json file (not a template) and then starts a build, using the supplied oc/cli client for both operations; ginkgo error checking included
+func StartBuildFromJSON(jsonFile, buildPrefix string, oc *CLI) {
```

Comment:
was no one calling this?  i'm surprised you didn't have to touch anything else when renaming it.